AppDashboard/dashboard.py

```diff
@@ -105,16 +105,21 @@ def render_template(self, template_file, values=None):
     if values is None:
       values = {}
 
-    owned_apps = self.dstore.get_owned_apps()
-    self.helper.update_cookie_app_list(owned_apps, self.request, self.response)
+    is_cloud_admin = self.helper.is_user_cloud_admin()
+    if is_cloud_admin:
+      apps_user_is_admin_on = self.dstore.get_application_info()
+    else:
+      apps_user_is_admin_on = self.helper.get_owned_apps()
```

Comment:
apps_user_is_admin on is a dictionary of the applications that is used in relocate, delete, and app stats. I'm fine with a rename but the appids are needed in all of those.